example/demo_survey/src/survey/widgets/home.tsx

```diff
@@ -385,6 +385,7 @@ export const homeApartmentNumber = {
   path: "home.apartmentNumber",
   datatype: "string",
   twoColumns: true,
+  joinPrevious: true,
```

Comment:
Interesting approach, but instead of joinPrevious, maybe use 'joinWith', which allows to specify which question to join it with. What if the previous one is actually invisible and it ends up being joined with the wrong other question?

But if so, that would complicate everything because it would need to be done at the Section level instead of Question

Reply:
I would suggest joinWith and if the joined with is not previous, just ignore

Reply:
I will try to see what I can do for join with. I made join with previous because questions will usually be hidden until an answers in that is before it is answered.